ibm/operator_collection_sdk/playbooks/molecule/create_redeploy_delete_operator/prepare.yml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2023 IBM Inc. All rights reserved
+# SPDX-License-Identifier: Apache2.0
+#
+
+---
+- name: Prepare
+  hosts: all
+  connection: local
+  gather_facts: false
+  vars:
+    ocpnamespace: "{{ ocpNamespace | lower | replace('_','-') | replace('/','-') }}"
+    
+  tasks:
+    - name: Validate OCP cluster authorization
+      ansible.builtin.command: oc project
+      register: oc_project_output
+      ignore_errors: true
+
+    - name: Cluster login
+      ansible.builtin.command: oc login --token={{ openshift_token }} --server={{ openshift_server }}
```

Comment:
Will this fail on users who have to enter the "--skip-insecure-tls-verify" flag? Or because this is molecule it'll only be run in a pipeline, not on a user machine